br/pkg/pdutil/pd.go

```diff
@@ -26,13 +26,15 @@ import (
 	"github.com/pingcap/tidb/br/pkg/lightning/common"
 	"github.com/pingcap/tidb/util/codec"
 	pd "github.com/tikv/pd/client"
+	pdapi "github.com/tikv/pd/server/api"
```

Comment:
Directly import PD components leads to flag pollution. PD sets `check.v` flag in some packages(not test package), however, TiDB is highly depend on the flags when bootstrapping variables(test and prod env has different variables).

More details can be found https://github.com/pingcap/tidb/issues/27156#issuecomment-900003955.